skilltree-tree/src/main.rs

```
@@ -34,19 +43,21 @@ fn main() {
     let mut skills: Vec<String> = vec![];
 
     for slice in sliced {
+        if slice.contains("span") {
+            println!("Element containing span might not be displayed");
+        }
+
```

Make this a panic instead of a println